src/cogs/error_handler.py

```diff
@@ -0,0 +1,105 @@
+"""
+A module that provides error handling using events.
+"""
+import logging
+import secrets
+
+import discord
+from discord.ext import commands
+
+
+class CommandErrorHandler(commands.Cog):
+    """A cog for handling command errors."""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    logging.basicConfig(
+        filename="error.log",
+        filemode="a",
+        format="%(asctime)s [%(levelname)s] %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+        level=logging.ERROR,
+    )
+
+    def generate_log_id(self):
+        """Generates a unique ID with secrets.token_hex()"""
+        return str(secrets.token_hex(4))
+
+    def create_error_string(self, ctx, error):
+        """Returns an error string depending on the error type."""
+        # change to "/" when that's implemented
+        prefix = "!"
```

Comment:
I'd get rid of this variable, and use `self.bot.prefix`. Don't define it in multiple places

Reply:
As far as I could see we didn't have `self.bot.prefix` defined anywhere, so I went ahead and did that in [this commit](https://github.com/pcparadise/discordbot/pull/49/commits/2c0100dc14b14235a49afe071571471ba11544b2). With that said we still need to switch `help.py`  to use the same global prefix method, instead of manually defining each one.

Reply:
Ah yeah, forgot it was in self.bot.config, I prefer it the way you just added in anyways.